src/itype.h

```diff
@@ -342,13 +342,19 @@ struct islot_wheel
 
         /** width of wheel (inches) */
         int width = 0;
+
+        /** coefficient of friction (percent) */
```

Comment:
The value it is replacing is `diameter*width`, so contact area. But here you use it as if it was unitless.